x-pack/legacy/plugins/ml/server/models/calendar/calendar_manager.ts

```diff
@@ -6,17 +6,46 @@
 
 import _ from 'lodash';
 import Boom from 'boom';
-import { EventManager } from './event_manager';
+import { EventManager, CalendarEvent } from './event_manager';
+
+interface BasicCalendar {
+  job_ids: string[];
+  description?: string;
+  events: CalendarEvent[];
+}
+
+export interface Calendar extends BasicCalendar {
+  calendar_id: string;
+}
+
+export interface FormCalendar extends BasicCalendar {
+  calendarId: string;
+}
 
 export class CalendarManager {
-  constructor(callWithRequest) {
-    this.callWithRequest = callWithRequest;
-    this.eventManager = new EventManager(callWithRequest);
+  private isLegacy: boolean;
+  private client: any;
```

Comment:
for recent classes that i've written i've started all private class members with a an underscore.
I don't know if this is something we want to try to do everywhere. I quite like the naming convention as it makes it easy to spot private and public members.
Other parts of Kibana have adopted this pattern but it seems to depend on the developer.

Reply:
Updated naming these with the underscore - e4c0e64639120c16c78f85258334091d4ec0c6ea